src/Microsoft.Spatial/GeoJsonWriterBase.cs

```diff
@@ -447,15 +447,8 @@ private void EndShape()
         /// <summary>
         /// Writes the coordinate reference system footer for the GeoJson object.
         /// </summary>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Globalization", "CA1303:Do not pass literals as localized parameters", MessageId = "Microsoft.Spatial.JsonWriter.WriteQuotedString(System.String)", Justification = "Values have no localized content.")]
         private void WriteCrs()
         {
-            // "crs": {

```

Comment:
why remove these comments codes? 

It's better to leave them unchanged to improve the understanding.